lib/commands/update.js

```diff
@@ -84,6 +84,13 @@ class UpdateCommand extends Command {
                 task: this.removeOldVersions
             }];
 
+            // CASE: We have moved the execution of knex-migrator into Ghost 2.0.0.
+            //       If you are already on ^2 or you update from ^1 to ^2, then skip the task.
+            if (semver.satisfies(instance.cliConfig.get('active-version'), '^2.0.0') ||
+                semver.satisfies(context.version, '^2.0.0')) {
+                tasks.splice(3, 1);
```

Comment:
easier way to do this is create an `enable` function in the migrate task, rather than splicing the array

Reply:
cool will try 👍